app/src/main/java/org/mozilla/fenix/tabstray/browser/AbstractBrowserTabViewHolder.kt

```diff
@@ -128,9 +129,8 @@ abstract class AbstractBrowserTabViewHolder(
     }
 
     private fun updateTitle(tab: TabSessionState) {
-        val title = tab.content.title.ifEmpty {
-            tab.content.url
-        }
+        // We can use the max URI length for titles as well.
+        val title = tab.toDisplayTitle().take(MAX_URI_LENGTH)
```

Comment:
I have no preference. I choose `take` because it was clear what I was trying to fix and aligned with examples a few lines below.

A linter would be nice if there is a stronger preference to use `trimmed`. Discovery might be hard otherwise?